lizard_auth_server/forms.py

```diff
@@ -4,7 +4,7 @@
 
 from django import forms
 from django.conf import settings
-from django.contrib import auth
+from django.contrib.auth import forms as authforms
```

Comment:
Ik weet niet precies waarom, maar ik krijg een AttributeError met `auth.forms.PasswordChangeForm` nu, en niet met `authforms.PasswordChangeForm`.
Met `from django.contrib.auth.forms import PasswordChangeForm` gaat het trouwens ook goed. Enig idee wat er anders is?